src/main/scala/com/microsoft/hyperspace/util/ConfigUtils.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.util
+
+import org.apache.spark.sql.SparkSession
+
+import com.microsoft.hyperspace.index.IndexConstants
+
+/**
+ * Helper class to extract Hyperspace config value
```

Comment:
Thanks for doing this. I was thinking of doing something similar.

Similar to [SQLConf](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala), can you rename this to `HyperspaceConf`?

And we can slowly move config constants into this class. (You don't need to do this in this PR).

cc: @apoorvedave1, @pirz

Reply:
Renamed. Thanks!